ironfish-cli/src/commands/wallet/transaction.ts

```diff
@@ -61,41 +63,54 @@ export class TransactionCommand extends IronfishCommand {
     if (response.content.transaction.notes.length > 0) {
       this.log(`---Notes---\n`)
 
-      CliUx.ux.table(response.content.transaction.notes, {
-        owner: {
-          header: 'Owner',
-          get: (note) => (note.owner ? `✔` : `x`),
+      CliUx.ux.table(
+        response.content.transaction.notes,
+        {
+          owner: {
+            header: 'Owner',
+            get: (note) => (note.owner ? `✔` : `x`),
+          },
+          amount: {
+            header: 'Amount',
+            get: (note) => CurrencyUtils.renderIron(note.value),
+          },
+          assetName: {
+            header: 'Asset Name',
+          },
+          assetId: {
+            header: 'Asset Id',
+          },
+          isSpent: {
+            header: 'Spent',
+            get: (note) => (!note.owner ? '?' : note.spent ? `✔` : `x`),
+          },
+          memo: {
+            header: 'Memo',
+          },
         },
-        amount: {
-          header: 'Amount',
-          get: (note) => CurrencyUtils.renderIron(note.value),
+        {
+          ...flags,
         },
-        assetName: {
-          header: 'Asset Name',
-        },
-        assetId: {
-          header: 'Asset Id',
-        },
-        isSpent: {
-          header: 'Spent',
-          get: (note) => (!note.owner ? '?' : note.spent ? `✔` : `x`),
-        },
-        memo: {
-          header: 'Memo',
-        },
-      })
+      )
     }
 
     if (response.content.transaction.assetBalanceDeltas) {
       this.log(`---Asset Balance Deltas---\n`)
-      CliUx.ux.table(response.content.transaction.assetBalanceDeltas, {
-        assetId: {
-          header: 'Asset ID',
+
+      CliUx.ux.table(
+        response.content.transaction.assetBalanceDeltas,
+        {
+          assetId: {
+            header: 'Asset ID',
+          },
+          delta: {
+            header: 'Balance Change',
+          },
         },
-        delta: {
-          header: 'Balance Change',
+        {
+          ...flags,
```

Comment:
Passing the flags to multiple tables will break for some flags, like filter:
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/57735705/213096534-500d0ca0-a691-49cf-95ff-cc0352febb32.png">